src/appmixer/ai/openai/AIAgent/AIAgent.js

```
@@ -97,47 +179,81 @@
+            content: prompt
+        });
+
+        for (let i = 0; i < context.config.AI_AGENT_MAX_ATTEMPTS || MAX_ATTEMPTS; i++) {
+
+            const response = await client.chat.completions.create({
+                model,
+                messages,
+                tools
+            });
+
+            const { finish_reason, message } = response.choices[0];
```

ESLint

[error] Identifier 'finish_reason' is not in camel case.

(camelcase)